### PR TITLE
Codepath implementation

### LINKED ISSUE #109: Prevent overlapping boundaries in code path report
Original issue: https://github.com/undefinedlabs/ui/issues/1603

As discussed with @fermayo, this is something that should be normalized in the agents and not the UI

### PR DESCRIPTION
**Pull request by tonyredondo**
Close #109

Implementation of the codepath exposing the private `cover` var with the linker, by running the test using:

```
go test -covermode=count -coverprofile=coverage.out -v ./...
```

**Limitation:**
- Parallel test execution is not supported.

**Demo code:**
https://github.com/undefinedlabs/demo-go/tree/codepath

**Results:**
https://shared.scope.dev/dashboard/8cee2622-8772-4338-8d72-63653e0e0239/66857b5a-cd46-4b9a-a6b3-438a51b84b4b/default/explore/inspect/8b105c3ee56fc7881e63f009c44b5fd59f32aca6/00000000-0000-0000-5a78-3c86d4e87f89/path

### CODE REVIEW

File: instrumentation/testing/coverage.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	for name, counts := range cover.Counters {
+		blocks := cover.Blocks[name]
+		for i := range counts {
+			stmts := int64(blocks[i].Stmts)
+			total += stmts
+			count = atomic.LoadUint32(&counts[i])
+			atomic.StoreUint32(&counts[i], counters[name][i]+count)
+			if count > 0 {
+				active += stmts
+				file := fmt.Sprintf("%v/src/%v", gopath, name)
```
Comment (fermayo) [inline instrumentation/testing/coverage.go]: Is this always true? With go modules you can have your code outside of `GOPATH` right?

Comment (tonyredondo) [reply instrumentation/testing/coverage.go]: Yes, you’re right, i have to implement the same algorithm used by go tool to find the files as you can see here:  https://github.com/golang/go/blob/master/src/cmd/cover/func.go#L178

findpkg and findfile. But they use internally go list to accomplish that...

Comment (tonyredondo) [reply instrumentation/testing/coverage.go]: Done, the algorithm is implemented.

Comment (tonyredondo) [reply instrumentation/testing/coverage.go]: https://shared.scope.dev/dashboard/8cee2622-8772-4338-8d72-63653e0e0239/66857b5a-cd46-4b9a-a6b3-438a51b84b4b/default/explore/inspect/8b105c3ee56fc7881e63f009c44b5fd59f32aca6/00000000-0000-0000-5a78-3c86d4e87f89/path

**Pull request merged**